packages/query/src/graphql/plugins/smartTagsPlugin.ts

```diff
@@ -8,7 +8,7 @@ export const smartTagsPlugin = makePgSmartTagsPlugin([
   {
     //Rule 1, omit `_metadata` from node
     kind: PgEntityKind.CLASS,
-    match: ({name}: PgEntity) => /^_metadata$/.test(name),
+    match: ({name}: PgEntity) => /^_metadata$/.test(name) || /^_metadata_[a-zA-Z0-9-]+$/.test(name),
```

Comment:
Can we define this regex in utils package? Its used in a few places, it would be good to have a single source of truth